core/lib/workarea/ext/freedom_patches/dragonfly_job_fetch_url.rb

```diff
@@ -1,6 +1,16 @@
 module Dragonfly
   class Job
     class FetchUrl < Step
+      class ProxyError < StandardError
+        def initialize(url)
+          super %(
+            Error fetching #{url}:
```

Comment:
open to any suggestions on the text here, I just kinda wrote this out as a first draft